src/python/bot/minimizer/minimizer.py

```diff
@@ -397,6 +396,7 @@ def _attempt_merge(self, hypotheses, sibling_merge_succeeded=False):
     back = hypotheses[middle:]
 
     # If we could remove either one of two hypotheses, favor removing the first.
+    # pylint: disable=assignment-from-none
```

Comment:
What is this ? @mbarbella-chromium fyi

Reply:
This I have to add because self._attempt_merge doesn't returns anything but according to the logic of the code, this should returns something. This is probably a bug, I confirmed with @oliverchang and have reported it here #1845

Reply:
Probably better to not add this pylint then as it is a real bug.

Reply:
If I removed this pyllint, the travis CI build will fail as I have made another change to this file. I can probably remove both of them and after fixing this bug we can change the other lint issue?

Reply:
ok fine to keep then. add another: 
FIXME: Fix this. Tracked in https://github.com/google/clusterfuzz/issues/1845.